CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## CraftDocker 1.1.14 ##
```

Comment:
Technically this is following [semantic versioning](http://semver.org/) only if craft is, i.e. if the craft update includes only bugfixes and no new features.